maths/binary_exponentiation.py

```diff
@@ -1,20 +1,78 @@
-"""Binary Exponentiation."""
+def binary_exponentiation_multiplication(a, b):
+    """
+    * Binary Exponentiation with Multiplication
+    * This is a method to find a*b in a time complexity of O(log b)
+    * This is one of the most commonly used methods of finding result of multiplication.
+    * Also useful in cases where solution to (a*b)%c is required,
+    * where a,b,c can be numbers over the computers calculation limits.
+    * Done using iteration, can also be done using recursion
 
-# Author : Junth Basnet
-# Time Complexity : O(logn)
+    * Let's say you need to calculate a ^ b
```

Comment:
```suggestion
    * Let's say you need to calculate a * b
```
The rest of the documentation says that this function calculates `a * b`, not `a ** b`